code/obj/machinery/deep_fryer.dm

```diff
@@ -27,6 +27,9 @@
 		if (isghostdrone(user) || isAI(user))
 			boutput(usr, "<span class='alert'>The [src] refuses to interface with you, as you are not a properly trained chef!</span>")
 			return
+		if (issilicon(user) && W.loc == user) //For borg held items
```

Comment:
Could probably check `W.cant_drop` instead? That's set on all cyborg tools, and handles a few weird things like kudzumen vines, ghostdrone tools (covered by check above in this case, but not always), item arms.

Reply:
Done!